### DISCUSSION TITLE
New Fleet Telemetry documentation

### DISCUSSION THREAD

**Discussion by pkuehnel** [teslamate-org/teslamate#3605] [Ideas]
It looks like Tesla published a Fleet Telemetry documentation [here](https://developer.tesla.com/docs/fleet-api#fleet_telemetry_config) and it includes a bunch of new fields see [*.proto files here](https://github.com/teslamotors/fleet-telemetry/tree/main/protos). There a many new fields, even things like `PackVoltage` or `ModuleTempMax`. Could be worth to look into it.

**Comment by KrzysztofHajdamowicz** [comment]
That would require migrating to FleetAPI first.

**Comment by micves** [comment]
There is already a lot going on about this in https://github.com/teslamate-org/teslamate/discussions/3792
